Content.Client/UserInterface/ActionMenuItem.cs

```diff
@@ -38,6 +43,12 @@ public ActionMenuItem(BaseActionPrototype action)
             TooltipSupplier = SupplyTooltip;
         }
 
+        protected override void ControlFocusExited()
+        {
+            base.ControlFocusExited();
+            _onControlFocusExited.Invoke(this);
```

Comment:
Should this be

```suggestion
            _onControlFocusExited?.Invoke(this);
```

Reply:
I have `#nullable enable` at the top, so pretty sure this check shouldn't be needed... this var would be a non-nullable reference type, thus the value should never be null